advanced/is-pattern-1/templates/is/wso2is-pattern-1-identity-server-bin.yaml

```diff
@@ -234,9 +234,9 @@ data:
     # ---------- Handle the SSL Issue with proper JDK version --------------------
     java_version=$("$JAVACMD" -version 2>&1 | awk -F '"' '/version/ {print $2}')
     java_version_formatted=$(echo "$java_version" | awk -F. '{printf("%02d%02d",$1,$2);}')
-    if [ $java_version_formatted -lt 0107 ] || [ $java_version_formatted -gt 1100 ]; then
-       echo " Starting WSO2 Carbon (in unsupported JDK)"
-       echo " [ERROR] CARBON is supported only on JDK 1.7, 1.8, 9, 10 and 11"
+    if [ $java_version_formatted -lt 1100 ] || [ $java_version_formatted -gt 1700 ]; then
+      echo " Starting WSO2 Carbon (in unsupported JDK)"
+      echo " [ERROR] CARBON is supported only between JDK 11 and JDK 17"
```

Comment:
Do we support JDK 15 and any other JDK version between 11 and 17? If not, adding the word `between` is not okay. We have to specify exact versions we support.